utils/http/errors.py

```diff
@@ -0,0 +1,12 @@
+class ResponseError(Exception):
```

Comment:
Rename to `HttpErrorResponseError` (I know, two `Error`s in the name -- the first is to denote that there is an HTTP error; the second is to denote that it is a special Python object which extends `Error`/`Exception`)

Reply:
Done